integration-tests/go.mod

```diff
@@ -233,6 +233,7 @@ require (
 	github.com/mimoo/StrobeGo v0.0.0-20210601165009-122bf33a46e0 // indirect
 	github.com/minio/blake2b-simd v0.0.0-20160723061019-3f5f724cb5b1 // indirect
 	github.com/minio/sha256-simd v0.1.1 // indirect
+	github.com/mitchellh/go-homedir v1.1.0 // indirect
```

Comment:
This is odd...no other go.mod changes to cause this?

Reply:
Nope, but it is definitely needed:

```
➜  integration-tests git:(develop) go mod why -m go.mod github.com/mitchellh/go-homedir
# go.mod
(main module does not need module go.mod)

# github.com/mitchellh/go-homedir
github.com/smartcontractkit/chainlink/integration-tests/actions
github.com/smartcontractkit/chainlink/v2/core/services/job
github.com/smartcontractkit/chainlink/v2/core/config
github.com/smartcontractkit/chainlink/v2/core/config.test
github.com/smartcontractkit/chainlink/v2/core/config/parse
github.com/mitchellh/go-homedir
```

Reply:
(It seems to have been caused by adding the parse.HomeDir call in the ValidateConfig)

Reply:
Ah I guess this module uses the config type packages.